docs/cloud-dev-env.md

```diff
@@ -0,0 +1,26 @@
+# Cloud Development Environments
+
+Altis has experimental built-in support for [GitHub Codespaces](https://github.com/features/codespaces), a Cloud-based development environment, that makes it possible to spin up a complete development environment in seconds within your browser, with a full editor based on Visual Studio Code and a fully-featured environment for your project, without having to install anything on your local machine.
+
+This is useful for many reasons:
+    - Few seconds to bootstrap a full development environment.
+    - Zero local installation required.
+    - Standardized and highly configurable environment setup.
+    - Powerful development environment, scalable as needed.
+    - Possibility of sharing preview environments with colleagues as needed.
+
+## How to start
+
+<!-- a note about pricing and associated logistics here? -->
+
+    - Ask Altis support to activate Codespaces feature for your project members.
```

Comment:
I think this only applies to repos under our organisation, we need instructions for both cases

Reply:
Yes, IF we're going to allow usage on our org, I'd like to wait to see Ryan's input on this to suit our existing expectations if any.